packages/wrangler/src/constellation/listModel.tsx

```diff
@@ -0,0 +1,66 @@
+import Table from "ink-table";
+import React from "react";
+import { fetchResult } from "../cfetch";
+import { withConfig } from "../config";
+import { logger } from "../logger";
+import { requireAuth } from "../user";
+import { renderToString } from "../utils/render";
+import { Name } from "./options";
+import { constellationBetaWarning, getProjectByName } from "./utils";
+import type {
+	CommonYargsArgv,
+	StrictYargsOptionsToInterface,
+} from "../yargs-types";
+import type { Project, Model } from "./types";
+
+export function Options(yargs: CommonYargsArgv) {
+	return Name(yargs)
+		.option("json", {
+			describe: "return output as clean JSON",
+			type: "boolean",
+			default: false,
+		})
+		.epilogue(constellationBetaWarning);
+}
+
+type HandlerOptions = StrictYargsOptionsToInterface<typeof Options>;
+export const Handler = withConfig<HandlerOptions>(
+	async ({ name, json, config }): Promise<void> => {
+		const accountId = await requireAuth(config);
+		const proj: Project = await getProjectByName(config, accountId, name);
+
+		const models: Array<Model> = await listModels(accountId, proj);
+
+		if (json) {
+			logger.log(JSON.stringify(models, null, 2));
+		} else {
+			logger.log(constellationBetaWarning);
+			logger.log(renderToString(<Table data={models}></Table>));
+		}
+	}
+);
+
+export const listModels = async (
+	accountId: string,
+	proj: Project
+): Promise<Array<Model>> => {
+	const pageSize = 10;
+	let page = 1;
+	const results = [];
+	while (results.length % pageSize === 0) {
+		const json: Array<Model> = await fetchResult(
+			`/accounts/${accountId}/constellation/project/${proj.id}/model`,
```

Comment:
Does this support a larger page size than `10`?

Reply:
Changed it to 50.